### DISCUSSION TITLE
Set default means of transport

### DISCUSSION THREAD

**Discussion by sphh** [osmandapp/OsmAnd#11998] [Q&A]
I use my bike more often than my car, so I want to have the route calculated for bikes by default. Unfortunately OSM+ always starts with calculating the route for cars.

I tried to reorder the means of transport and moved the bike to the first place. Unfortunately it did not help.

I could not find the option, where I can tell OSM+ that it should always use the bike route planner as a default. Could someone tell me, where I would find it, please?

**Comment by vshcherb** [comment]
You could change default profile in OsmAnd Settings, then you will always use application with bike profile

**Reply by sphh** [reply]
Where would I find that setting?

What I have:
1. In `Settings > Edit profile list` I reordered the list of profiles to the following: `Browse map`, `Cycling`, `Walking`, `Public transport`, `Driving`, …
2. In `Settings > OsmAnd settings > Default profile` I set `Browse map` as default profile, because whenever I open the app, I want this map to be shown.

But if I press the `Route` icon on the map, the find route screen opens with `Driving` selected at the top. I want `Cycling` to be pre-selected, when I open the find route screen.

How can I do this?

**Reply by Lee-Carre** [reply]
> Where would I find that setting?

The № 2 in your list.

> How can I do this?

If you want to keep that as the generic Browse Map, then I dunno; I encounter the same problem.

Maybe disable profiles which you don't use often?
